week1/3-website/index.html

```diff
@@ -18,5 +18,178 @@
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+      <header id="header">
+        <h1>SEASON OF MIGRATION TO THE         NORTH</h1>
+        <p id="prag">"The story of a man undone by a culture that in part created him,<br> Season of Migration to the North, is a powerful and evocative<br> examination of colonization in two vastly different worlds".</p>
+        <figure>
+          <img id="img-caption" src="https://images.randomhouse.com/cover/700jpg/9781590173022" alt="Book Cover">
+          <figcaption>
+            Book Cover
+          </figcaption>
+        </figure>
+      </header>
+      <section>
+        <article id="article">
```

Comment:
Three article elements with identical 'article' IDs? Never do this. You can either use a class, or apply CSS directly to the article element.